esp_idf/spotflow/include/configs/spotflow_config_cbor.h

```diff
@@ -0,0 +1,45 @@
+#ifndef SPOTFLOW_CONFIG_CBOR_H
+#define SPOTFLOW_CONFIG_CBOR_H
+
+#include <stdbool.h>
+#include <stdint.h>
+#include <stddef.h>
+
+#define SPOTFLOW_CONFIG_RESPONSE_MAX_LENGTH 32
+
+#ifdef __cplusplus
+extern "C" {
+#endif
+
+// Flags for desired
+#define SPOTFLOW_DESIRED_FLAG_MINIMAL_LOG_SEVERITY (1 << 0)
+
+// Flags for reported
+#define SPOTFLOW_PERSISTED_SETTINGS_FLAG_SENT_LOG_LEVEL (1 << 0)
```

Comment:
Please move this flag to `spotflow_config_persistence.h`, because persistence is unrelated to CBOR encoding. Furthermore, it's now unclear which flags can be used in the field `uint8_t flags` of `struct spotflow_config_persisted_settings`.